.gitignore

```diff
@@ -37,25 +39,136 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
+.nox/
 .coverage
 .coverage.*
 .cache
 nosetests.xml
 coverage.xml
-*,cover
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+cover/
 
 # Translations
 *.mo
 *.pot
 
 # Django stuff:
 *.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Scrapy stuff:
+.scrapy
 
 # Sphinx documentation
 docs/_build/
 
 # PyBuilder
+.pybuilder/
 target/
 
-# VS Code
-.vscode/
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# IPython
+profile_default/
+ipython_config.py
+
+# pyenv
+#   For a library or package, you might want to ignore these files since the code is
+#   intended to run in multiple environments; otherwise, check them in:
+.python-version
```

Comment:
@Kludex , should it be commented or not? There is an [answer](https://stackoverflow.com/questions/54315206/should-we-gitignore-the-python-version-file), but your decision will be final.

Reply:
You can add the `.python-version` to the gitignore, yes.

But I don't think we need to add that many things to the `.gitignore`, only add when things are needed.

Reply:
Which would you like to keep here definitely? I think I can remove other project management packages, but what else?